tests/integrated/test-laplace/runtest

```diff
@@ -52,7 +52,7 @@ for v in vars:
 print("Running Laplacian inversion test")
 success = True
 
-for solver in ["cyclic", "pcr"]:
+for solver in ["pcr", "cyclic", "pcr_or_cyclic"]:
```

Comment:
Not really sure adding this to this test is useful -- it's always PCR. The grid file would need changing such that PCR was sometimes not acceptable, but I think that basically necessitates an entirely new test, or _only_ using `pcr_or_cyclic`